src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerServiceTest.java

```diff
@@ -1069,6 +1069,13 @@ void testCaseDeepLink() {
         assertEquals("http://domain/cases/case-details/123#Documents", caseData.getCaseDeepLink());
     }
 
+    @Test
+    void testHearingIsLinkedFlag() {
+        CaseData caseData = new CaseData();
+        caseManagementForCaseWorkerService.setHearingIsLinkedFlag(caseData);
+        assertEquals(Boolean.FALSE.toString(), caseData.getHearingIsLinkedFlag());
+    }
+
```

Comment:
Do we need a test here as well checking that it sets the flag to true/Yes ?

Reply:
The caseManagementForCaseWorkerService only adds the flag and defaults it to false.
The createCaseLinkAboutToSubmit method (on CaseLinksController) changes it to yes.